### PR TITLE
feat: added react router

### PR DESCRIPTION
**Pull request by Digambar1411**
Added routes to all pages

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,24 @@
...
-        </div>
-      </header>
+      <Routes>
+        <Route path= "/home" element = { <Home/> }/>
+        <Route path= "/login" element = { <Login/> }/>
+        <Route path= "/signup" element = { <Signup/> }/>
+        <Route path= "/cart" element = { <Cart/> }/>
+        <Route path= "/wishlist" element = { <Wishlist/> }/>
+        <Route path= "/products" element = { <Products/> }/>
+      </Routes>
```
Comment (Naveenchandar) [inline src/App.js]: You can move routes as separate component. It will be useful in future if many routes are added.

**Review by Naveenchandar — APPROVED** [review]
Good folder structure.

File: src/App.js (hunk 1 above)
Comment (maitrakhatri) [inline src/App.js]: Add route for 404 page

**Review by maitrakhatri — APPROVED** [review]
Good folder structure.
Good use of barrel exports
Just make sure your component file names start with Capital Letters 
Left a suggestion

File: src/App.js (hunk 1 above)
Comment (Digambar1411) [reply src/App.js]: as of now i will keep same. for next project will do the same

Comment (Digambar1411) [reply src/App.js]: yeah thanks. it has been added

**Pull request merged**